src/schema/mixs.yaml

```diff
@@ -7,7 +7,8 @@ notes:
 id: https://raw.githubusercontent.com/microbiomedata/nmdc-schema/main/src/schema/mixs.yaml
 imports:
   - linkml:types
-  - attribute_values
+  - attribute_values_classes
```

Comment:
that change has to be configured in a helper file

Reply:
helper file?

Reply:
`assets/yq-for-mixs_subset_modified.txt`

```
'.imports += ["attribute_values"]'
```

Reply:
I just changed the name of the file back to attributes_values.yaml -- it didn't need to be changed.  The name change was from the initial attempt to separate classes from slots.  At the end of the day, we decided not to do this separation but the file name stuck.